src/Platform/Microsoft.Testing.Platform.MSBuild/Tasks/TestingPlatformEntryPointTask.cs

```diff
@@ -135,11 +135,11 @@ internal sealed class TestingPlatformEntryPoint
 <System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage>
 Module TestingPlatformEntryPoint
 
-    Function Main(args As Global.System.String()) As Global.System.Int32
-        Return MainAsync(args).Result
+    Function Main(args As String()) As Integer
+        Return MainAsync(args).GetAwaiter().GetResult()
     End Function
 
-    Public Async Function MainAsync(ByVal args() As Global.System.String) As Global.System.Threading.Tasks.Task(Of Integer)
+    Public Async Function MainAsync(args As String()) As Global.System.Threading.Tasks.Task(Of Integer)
```

Comment:
- `ByVal` is redundant. Parameters are `ByVal` by default.
- `()` can be put either after parameter name or after type and both are exactly the same and equivalent. Few lines above we already used the style of putting it after the type, so I switched this one to match.